src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -516,5 +517,16 @@
 							"	cancel event")
 					.since("2.3");
 		}
+		if (Skript.classExists("com.destroystokyo.paper.event.entity.EntityPathfindEvent")) {
+			Skript.registerEvent("Pathfind", SimpleEvent.class, EntityPathfindEvent.class, "[entity] path[ ]find[ing]")
+					.description("Called whenever an entity decides to start moving to a location.",
+							"This event requires PaperSpigot.")
```

Comment:
```suggestion
```
requiredPlugins is enough

Reply:
Should probably standardize this with the other SimpleEvents, I've seen a lot of them using "This event requires PaperSpigot." in the description instead of the requiredPlugins

Reply:
Because the requiredPlugins is added later